src/main/java/graphql/parser/GraphqlAntlrToLanguage.java

```diff
@@ -918,4 +927,11 @@ private List<Type> getImplementz(GraphqlParser.ImplementsInterfacesContext imple
         }
         return implementz;
     }
+
+    private <T extends Node<?>> T captureRuleContext(T node, ParserRuleContext ctx) {
```

Comment:
Maybe a comment somewhere that says if the map is null it means its not used.  For future devs.